### PR TITLE
fix: avoid top-level await

### PR DESCRIPTION
**Pull request by KnorpelSenf**
Dropping the only instance of TLA

### CODE REVIEW

**Review by KnightNiwrem — COMMENTED** [review]
Pending

https://t.me/grammyjs/225486
https://github.com/denoland/deploy_feedback/issues/527

File: src/platform.deno.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ const DEBUG = "DEBUG";
 if (isDeno) {
     debug.useColors = () => !Deno.noColor;
     const env = { name: "env", variable: DEBUG } as const;
-    const res = await Deno.permissions.query(env);
+    const res = Deno.permissions.querySync(env);
     let namespace: string | undefined = undefined;
     if (res.state === "granted") namespace = Deno.env.get(DEBUG);
```
Comment (wojpawlik) [inline src/platform.deno.ts:13]:
```suggestion
    const res = Deno.permissions.querySync?.(env);
    let namespace: string | undefined = undefined;
    if (res?.state === "granted") namespace = Deno.env.get(DEBUG);
```

to accomodate Deno Deploy

**Comment by KnorpelSenf** [comment]
This is more complicated than I thought, and the benefits are small. I will abandon this.

**Comment by KnightNiwrem** [comment]
What happened?

**Comment by KnorpelSenf** [comment]
TLA is not supported on all platforms, `querySync` is not supported on all platforms, and if we migrate to JSR, we will have to unify all the platform-specific code anyway. I don't see a lot of value in figuring out a good short-term solution here.

**Pull request closed without merging**